Layout improvements and code clean up

rafmudaf opened the pull request:
This pull request fixes some problems we had in the layout of the sidebar and how it interacts with the content area. I've also done a bit of restructuring.

Please review the changes and lets discuss before merging.

rafmudaf committed:
Clean up imports

rafmudaf committed:
Consolidate navigation components

rafmudaf committed:
Improve the sidebar html layout

Removes the z component of the sidebar, lets bootstrap handle sizing

rafmudaf committed:
Consolidate home and navigation components

rafmudaf committed:
Fix layout issues in Model Builder main page

rafmudaf committed:
Add content to the home page, include json import

rafmudaf committed:
Clean up

rafmudaf committed:
Bug fix: include updated element id

lizzjs reviewed (changes requested):
Remove the hamburger icon to simplify layout

rafmudaf committed:
Remove sidebar collapse feature

The collapse was implemented by moving elements directly and this caused issues with filling the page for various size windows.
